src/js/actions/tools.js

```diff
@@ -985,21 +1000,28 @@ define(function (require, exports) {
             key: utilShortcuts.GLOBAL.TOOLS.MASK_SELECT,
             modifiers: {},
             fn: _vectorMaskHandler,
-            name: "Vector Mask"
+            name: "Vector Mask Mode"
         });
 
         _vectorSelectMaskHandler = function () {
             if (toolStore.getVectorMode()) {
-                flux.actions.tools.select(toolStore.getToolByID("superselectVector"));
+                flux.actions.tools.select(toolStore.getToolByID(directSelectToolID));
             }
         }.bind(this);
 
-        shortcutSpecs.push({
-            key: utilShortcuts.GLOBAL.TOOLS.VECTOR_SELECT,
-            modifiers: {},
-            fn: _vectorSelectMaskHandler,
-            name: "Super Select Vector"
-        });
+        if (toolStore.getVectorMode()) {
+            _vectorSelectMaskHandler = function () {
+                flux.actions.select(toolStore.getToolByID(directSelectToolID));
+            }.bind(this);
+
+            shortcutSpecs.push({
+                key: utilShortcuts.GLOBAL.TOOLS.VECTOR_SELECT,
+                modifiers: {},
+                id: directSelectToolID,
+                fn: _vectorSelectMaskHandler,
+                name: "Direct Selection Tool"
```

Comment:
If so, then don't forget about Vector Mask Mode above.  What are these shortcut names used for...?